resources/assets/js/pages/Thread.vue

```diff
@@ -32,6 +33,14 @@
                 this.locked = ! this.locked;
             },
 
+            togglePin () {
+                let uri = `/pinned-threads/${this.thread.slug}`;
```

Comment:
uri can be a const

Reply:
what would make little sense at the moment because of the `toggleLock` method it is done the same way.
I would try to stay consistent.

Reply:
Agreed @concept-core I'm trying to keep style and formatting consistent to minimize the changes in the diffs. Although, I do somewhat agree with @fbcarpinato in the general case.